### DISCUSSION TITLE
大型洲际导弹项目"V2"正式开始接受预定

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2919] [其他]
![image](https://image.gcores.com/f3e4d17d-7d37-4b13-a2b1-39fc835bf4bb.png?x-oss-process=image/quality,q_90/format,webp/watermark,image_d2F0ZXJtYXJrLnBuZw,g_se,x_10,y_10)
# "Pendragon"MIRV型弹道导弹 简称"V2"
# 其射程最大可达约21000KM
## 具有末端反导技术 可以在三门近防炮和激光系统拦截下成功命中目标
# 可以录入”萝卜“数据 在空洞内发射并精确打击空洞外目标
# 该导弹可以在地面设施的操纵下停留在近地轨道随时待命
# 可以使用ADF,ADA系列战斗机远程遥控发射
## 有意向的组织可以在该贴下留言
# 定金:15400000丁尼

*该武器不售卖给任何个人
*提供软件调试 但不提供发射设施及弹头
*创立者工业不对任何因用户操作不当导致的法律风险和人身安全负责

**Comment by mldlhg** [comment]
这个委托我接了

**Reply by I-ikun** [reply]
6

**Comment by MLx-x** [comment]
请问贵公司能提供更详细的技术参数吗，还有在这里可以参加无人机抽奖吗<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3d1687e289addc7611eba2c8585f4a9f_5782304423493346054.png" alt="丽都漫步-安比" width="75" >

**Reply by GrunderIndustries** [reply]
参数均已撰写在图示中

**Comment by Samu1KU** [comment]
我是学生，送我

**Comment by LLLingYu** [comment]
我是初生，倒贴我50

**Comment by Bicjet** [comment]
前来考古，估计为第一篇绳网用户帖

**Comment by IXfromThorne** [comment]
希望分配给空洞特勤组的资金足够……
实在不行我再接几个单子去
